rclcpp/include/rclcpp/executor.hpp

```diff
@@ -82,6 +82,18 @@ class Executor
   virtual void
   spin() = 0;
 
+  /**
+   * \sa rclcpp::Executor:spin() for more details
+   * \throws std::runtime_error when spin() called while already spinning
+   * @param exception_handler will be called for every exception in the processing threads
```

Comment:
We don't use `\` everywhere (some other `@` have slipped through reviews), but we should tend towards the desired state when possible, which is `\` everywhere. I'll fix these up in my version of this pr.